pkg/util/util.go

```diff
@@ -84,6 +84,17 @@ func FileExists(filename string) (bool, error) {
 	return true, nil
 }
 
+// Verify block device is not in use by calling openFile with O_EXCL flag.
+// Returns nil if device not in use, but a non-nil return value means
+// that either the device is in use or another error occurred during OpenFile.
+func CheckDeviceInUseBySystem(pathname string) error {
```

Comment:
@ciwang @saad-ali 
Why not call have this routine return a bool? It could be more of a check that way. Am I mistaken but you cannot easily distinguish if the call encountered an error if the device is in use? Then one can use it for both cases as a test.

Something like "func DeviceOpened(pathname string) (bool, error) {"


Also do we want to check to see if the path passed in is a device and return an error if not? Check out: 
[os.Stat](https://golang.org/pkg/os/#File.Stat) and it returns a [FileInfo](https://golang.org/pkg/os/#FileInfo) you can use to determine if it's a device or not. Probably worth logging a warning/error in the case of someone passing in a non device path tot his routine.

Assume you are getting bad input and make it easy for someone to know.